src/core/qgsrendercontext.cpp

```diff
@@ -30,12 +32,17 @@ QgsRenderContext::QgsRenderContext()
   : mFlags( DrawEditingInfo | UseAdvancedEffects | DrawSelection | UseRenderingOptimization )
 {
   mVectorSimplifyMethod.setSimplifyHints( QgsVectorSimplifyMethod::NoSimplification );
+  QgsDistanceArea distanceArea;
```

Comment:
Yes, it wrong, it a hack until the cause of the above problem is resolved
* loss of the set Ellipsoid values between setEllipsoid and measureLineProjected